python/hsfs/hopsworks_udf.py

```diff
@@ -916,7 +916,7 @@ def transformation_statistics(
     def output_column_names(self, output_col_names: Union[str, List[str]]) -> None:
```

Comment:
Changed the key name for `on_demand_transformation_function` to use property in `training_dataset_feature`. 
Changed the key name `name` in to use property in `hopswrks_udf`

Reply:
I mean should `transformation_function` not be `transformationFunction`?

Reply:
Aah yes you are right. Since `trainingDatasetFeatures` were always getting created from the backend it was working when I was testing. I changed it to camel case.